styles.css

```diff
@@ -13,6 +13,149 @@
     text-decoration: none;
   }
 
+  /* Sidepanel */
+
+  #mySidepanel {
+    background-color: #3c3a39;
+    height: 100%;
+  }
+
+  .sidepanel {
+    width: 0;
+    height: 100%;
+    position: fixed;
+    z-index: 1;
+    overflow-x: hidden;
+    transition: 0.5s;
+  }
+
+  .sidepanel a {
+    padding: 5px 8px 5px 8px;
+    text-decoration: none;
+    font-size: 25px;
+    color: #67798e;
+    transition: 0.3s;
+  }
+
+  .sidepanel .closebtn {
+    top: 0;
+    right: 25px;
+    font-size: 36px;
+  }
+
+  .openbtn {
+    font-size: 20px;
+    cursor: pointer;
+    background-color: #3c3a39;
+    color: #fff;
+    padding: 10px 15px;
+    border: none;
+  }
+
+  .openbtn:hover {
+    background-color: royalblue;
+  }
+
+  .pan-header {
+    height: 72px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .pan-content {
+    height: auto;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-left: 70px;
+  }
+
+  .pan-scoreboard {
+    height: 35px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    top: 0;
+    left: 0;
+  }
+
+  .pan-watch {
+    border-top: 40px;
+    border-right: 40px;
+    height: 20px;
+    width: 30px;
+    top: 8px;
+    left: 30px;
+    margin-left: 70px;
+  }
+
+  .pan-board {
+    border-top: 20px;
+    height: 35px;
+    width: 80px;
+    left: 0;
+    border-radius: 0;
+    margin-right: 50px;
+    display: flex;
+    flex-direction: row;
+    justify-content: left;
+  }
+
+  .pan-burger {
+    height: 35px;
+    margin-right: 50px;
+    display: flex;
+    flex-direction: row;
+    justify-content: right;
+  }
+
+  .pan-content a {
+    width: 327px;
+    height: 70px;
+  }
+
+  .pan-content img {
+    height: 10px;
+    width: 327px;
```

Comment:
- [Optional] It'd be better if this line had a % based width like 85%, so it looks consistent on mobile screens of different sizes

Reply:
Thank you, Leonardo. I will take this into account.